api/HTMLTextAreaElement.json

```diff
@@ -143,27 +143,26 @@
           }
         }
       },
-      "labels": {
```

Comment:
This "labels" entry was only moved in this PR, no actual changes here. I've confirmed that's the case for everything that appears modified in the diff, so it can be safely ignored.